pytestqt/qtbot.py

```diff
@@ -234,7 +234,7 @@ def stopForInteraction(self):
 
     stop = stopForInteraction
 
-    def waitSignal(self, signal=None, timeout=1000, raising=None):
+    def waitSignal(self, signal=None, timeout=1000, callback=None, raising=None):
```

Comment:
`check_params_cb` maybe? I think `_cb` is a common suffix, and `check_params_callback` seems so long :wink: